### PR TITLE
Prepare release 1.0

### PR DESCRIPTION
**Pull request by Nyholm**
Just a clarification in the changelog.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     ],
     "extra": {
         "branch-alias": {
-            "dev-master": "0.10-dev"
+            "dev-master": "1.1-dev"
```
Comment (dbu) [inline composer.json]: we did not tag 1.0.0 - i would only change this once we add a new feature to discovery.

Comment (Nyholm) [inline composer.json]: Okey, I thought you changed this before you made the tag. Sorry about that.

Comment (sagikazarmark) [inline composer.json]: AFAIK this should be changed to 1.0-dev as we did in other packages

Comment (dbu) [inline composer.json]: ups, yes. i agree. sorry, somehow missed that its currently 0.10 and not 1.0

**Pull request merged**